### PR TITLE
Run tests for calicoctl, pod2daemon, and app-policy

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -364,24 +353,58 @@ blocks:
...
+  dependencies: []
+  task:
+    prologue:
+      commands:
+      - cd app-policy
+    jobs:
+    - name: "tests"
+      commands:
+      - make ci
+
```
Comment (robbrockbank) [inline .semaphore/semaphore.yml:400]: also missing CNI plugin

**Review by robbrockbank — APPROVED** [review]
LGTM

**Comment by caseydavenport** [comment]
Tests are failing, working on it...

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Pull request merged**